libhb/enc_qsv.c

```diff
@@ -309,8 +309,8 @@ static int qsv_hevc_make_header(hb_work_object_t *w, mfxSession session)
     mfxU16 Height            = pv->param.videoParam->mfx.FrameInfo.Height;
     mfxU16 Width             = pv->param.videoParam->mfx.FrameInfo.Width;
     frameSurface1.Info       = pv->param.videoParam->mfx.FrameInfo;
-    frameSurface1.Data.VU    = av_mallocz(Width * Height / 2);
-    frameSurface1.Data.Y     = av_mallocz(Width * Height);
+    frameSurface1.Data.Y     = av_mallocz(Width * Height * 3 / 2);
```

Comment:
Ah, yes, obviously. :smile: For some reason, I was expecting 1 + 0.5 + 0.5 (i.e. 4:2:2).